Lib/fontTools/misc/fixedTools.py

```diff
@@ -130,40 +203,74 @@ def strToFixed(string, precisionBits):
 
 
 def strToFixedToFloat(string, precisionBits):
-	"""Convert a string to a decimal float, by first converting the float to a
-	fixed-point number with precisionBits fractional binary digits.
+	"""Convert a string to a decimal float with fixed-point rounding.
+
+	This first converts string to a float, then turns it into a fixed-point
+	number with ``precisionBits`` fractional binary digits, then back to a
+	float again.
+
 	This is simply a shorthand for fixedToFloat(floatToFixed(float(s))).
 
-	>>> import math
-	>>> s = '-0.61884'
-	>>> bits = 14
-	>>> f = strToFixedToFloat(s, precisionBits=bits)
-	>>> math.isclose(f, -0.61883544921875)
-	True
-	>>> f == fixedToFloat(floatToFixed(float(s), precisionBits=bits), precisionBits=bits)
-	True
+	Args:
+		string (str): A string representing a decimal float.
+		precisionBits (int): Number of precision bits.
+
+	Returns:
+		float: The transformed and rounded value.
+
+	Examples::
+
+		>>> import math
+		>>> s = '-0.61884'
+		>>> bits = 14
+		>>> f = strToFixedToFloat(s, precisionBits=bits)
+		>>> math.isclose(f, -0.61883544921875)
+		True
+		>>> f == fixedToFloat(floatToFixed(float(s), precisionBits=bits), precisionBits=bits)
+		True
 	"""
 	value = float(string)
 	scale = 1 << precisionBits
 	return otRound(value * scale) / scale
 
 
 def floatToFixedToStr(value, precisionBits):
-	"""Convert float to string using the shortest decimal representation (ie. the least
-	number of fractional decimal digits) to represent the equivalent fixed-point number
-	with 'precisionBits' fractional binary digits.
+	"""Convert float to string with fixed-point rounding.
+
+	This uses the shortest decimal representation (ie. the least
+	number of fractional decimal digits) to represent the equivalent
+	fixed-point number with ``precisionBits`` fractional binary digits.
 	It uses fixedToStr under the hood.
 
 	>>> floatToFixedToStr(-0.61883544921875, precisionBits=14)
 	'-0.61884'
+
+	Args:
+		value (float): The float value to convert.
+		precisionBits (int): Number of precision bits, *up to a maximum of 16*.
+
+	Returns:
+		str: A string representation of the value.
+
 	"""
 	fixed = otRound(value * (1 << precisionBits))
 	return fixedToStr(fixed, precisionBits)
 
 
 def ensureVersionIsLong(value):
-	"""Ensure a table version is an unsigned long (unsigned short major,
-	unsigned short minor) instead of a float."""
+	"""Ensure a table version is an unsigned long.
+
+	OpenType table version numbers are expressed as a single unsigned long
+	comprising of an unsigned short major version and unsigned short minor
+	version. This function detects if the value to be used a version number
+	looks too small, and converts it to fixed-point if so.
```

Comment:
What counts as too small? This should be documented because `0x0000_0064` is apparently too small, though someone might intend that to represent version 0.100.

How does it convert to fixed-point? This should be documented because someone might expect version 1.1 to be `0x0001_0001` or `0x0001_199A` or `0x0001_1000`.